### ISSUE TITLE
Allow Accessibility Focus to be Changed Programatically

### ISSUE THREAD

**Issue by webtfdml** [flutter/flutter#49072]
## Use case

I am using multiple stacks in my Flutter application. In some instances, I'm using a StreamBuilder to create a new widget as an overlay on the old widget. 

The problem is that I can't update the accessibility focus to go to the widget at the top of the stack. 

I've tried using FocusNode, but this doesn't work. All FocusNode does is allow me to change the input focus, but the accessibility focus remains the same. 

I've verified this by doing print statements. It states that the focus has been updated, but the accessibility focus remains on the widgets in the stack below. 

I'm not the first person to run into this problem. Other's have posted about this on StackOverflow  without a satisfactory solution. 

https://stackoverflow.com/questions/57273232/move-accessibility-focus-programmatically

https://stackoverflow.com/questions/53456758/flutter-shift-focus-to-specific-widget/59792181#59792181

## Proposal

When declaring a Semantic widget, I should be able to define a "takes accessibility focus" property. If there are multiple nested semantic widgets with this property set to true, the one deepest in the widget tree should take precedence.

**Comment by Ayoubdesigner101** [comment]
#49111

**Comment by shahryar-cmyk** [comment]
> #49111

I don't understand the reference you provide can you explain it how my issue is related to this one.

**Comment by webtfdml** [comment]
My specific use case can be resolved with the BlockSemantics widget. Disregard.
